tRecorderApi/api/views/stitch_takes.py

```diff
@@ -14,10 +14,7 @@ class SourceStitchView(views.APIView):
 
     def post(self, request):
         data = request.data
-        if "language" in data
-        and "version" in data
-        and "book" in data
-        and "chapter" in data:
+        if "language" in data and "version" in data and "book" in data and "chapter" in data:
             chunks = Take.stitchSource(data)
```

Comment:
method name convention

Reply:
I did not work with this file, it was throwing me an error because of the format being used...